src/D2L.CodeStyle.Analyzers/Helpers/WhitelistHelper.cs

```diff
@@ -0,0 +1,83 @@
+﻿using System;
+using System.Collections.Immutable;
+using System.IO;
+using System.Linq;
+using Microsoft.CodeAnalysis;
+using Microsoft.CodeAnalysis.Diagnostics;
+using Microsoft.CodeAnalysis.Text;
+
+namespace D2L.CodeStyle.Analyzers.Helpers {
+
+	internal static class WhitelistHelper {
+
+		public static WhitelistHelper<INamedTypeSymbol> TypeLevel(

```

Comment:
What's another `T` you have in mind?
The name `TypeLevel` didn't make any sense to me without reading this function.

Reply:
Figured the assembly-level whitelist I'm replacing here coooould be desirable at some point?